Adding README.md as this app has additional extensions to be installed

aiamuzz opened the pull request:
If you could get the flathub maintainers to add the additional 'Extension install command' before the 'Application Install command' ... it would help the users better, in fact the flathub application webpage is the right place for these instructions ... as not every user may find a need to visit the github page of the application.

aiamuzz committed:
Adding README.md as this app has additional extensions to be installed

aiamuzz committed:
Update README.md

aiamuzz committed:
Update README.md

AsciiWolf commented on README.md:
```diff
@@ -0,0 +1,23 @@
+Install and run Windows software on Linux
+
+
+## Installation Instruction :
```
Typo: Space before the colon. Also, I think that it should be "Instructions", not "Instruction".

aiamuzz committed:
Update README.md

aiamuzz committed:
Update README.md

AsciiWolf commented on README.md:
```diff
@@ -0,0 +1,23 @@
+Install and run Windows software on Linux
+
+
+## Installation Instruction's :
```
Instructions, not Instruction's.

AsciiWolf commented on README.md:
````diff
@@ -0,0 +1,23 @@
+flatpak install flathub org.phoenicis.playonlinux
+```
+
+## Run :
````
What is the point of spaces before colons?

aiamuzz replied:
is that so important ?
grammer and typo's i can understand ... but simple formatting ... is that so  much of a concern ?
I am no coder ... but will a space cause any problem in the functionality of the application ?

AsciiWolf replied:
@aiamuzz It looks bad and unprofessional.

aiamuzz replied:
@AsciiWolf ... seriously ... now we are going to debate over it !!!
bad & unprofessional ... since when did layout / formatting become unprofessional ... 

![deepinscreenshot_select-area_20190124130952](https://user-images.githubusercontent.com/3283676/51662338-83233e00-1fd9-11e9-9004-dfcb99e222d9.png)

If you tell me why there are spaces between the `+` and the text  ? 
then i'll tell you why the space precedes the colon ...

Dude ... its a simple README file ... your comments about why spaces here why spaces there is a little overboard ... wasting your time and mine ...

AsciiWolf replied:
If there is something "little overboard", it's your behavior. I was just trying to help fix the issues. That's what PR reviews are for. So, please, calm down.

Erick555 replied:
> If you tell me why there are spaces between the + and the text ?

There are no any spaces there. You may look at document preview here: https://github.com/flathub/org.phoenicis.playonlinux/blob/f54bb34d3cca7bc0710d97bb8093d95b5638bc49/README.md

> Dude ... its a simple README file ... your comments about why spaces here why spaces there is a little overboard ... wasting your time and mine ...

With such attitude you are wasting everybody time with this PR as no one will accept work from someone who refuses to take even trivial advice. Please reconsider this.

aiamuzz replied:
helping fix langauage/grammer is sure welcome and i did acknowledge it ... but don't you think you getting stuck on a 'space' is overboard ? ...

for one you certainly didn't answer my question, on why there are spaces on both sides of the '+' symbol(in the image i have posted above) ... care to answer it ?

barthalion replied:
@aiamuzz You are being extremely unhelpful. English isn't my mother tongue either and I always appreciate if someone points out any mistakes I made. There is no reason to get so emotional about it.

Erick555 replied:
I agree this is petty much overboard as removing that space takes a second and nobody expected any dispute about this.

I already answered your question. There are no single _space_ character in position you mentioned. Seeing any gap there which comes from web page formatting doesn't mean there is a _space_ (I hope you don't format your documents with spaces).

alexlarsson replied:
The space between "+" and the contents is just how github renders unified diff. If you don't like this, bring this up with github. It has nothing to do with flatpak.

aiamuzz replied:
@Erick555 ... Its not about attitude ... its about accepting what is a worthy feedback ... and what isn't ... and ignoring the isn't ...

@AsciiWolf ... spoke about the space once, i considered it for a little while, then thought, it wasn't significant enough to consider ... so i ignored it ... then he raised the same issue of 'space before a colon' again with a remark !!!

Firstly this is a simple layout & formatting of text ... not really of any importance to the application functionality ... that we are having such a deep debate about it ...

Secondly ever heard of the term 'kerning' ... google it ... you'll know the reason ... i design fonts as a profession, for me spacing between glyphs is important ... so in my professional capacity if i took a call on the spacing feedback and thought it was best to ignore ...

and when somebody continues to harp on the same thing or remark about it ... things do tend to blow up a little !!!

aiamuzz replied:
> The space between "+" and the contents is just how github renders unified diff. If you don't like this, bring this up with github. It has nothing to do with flatpak.

wow ... @alexlarsson ... hahahaha ... my question about the "+" was not an objection but more of a way to answer to @AsciiWolf 's question about spacing ... 

exactly ... its the way github renders it or the person who design that UI formatted it in that way, as he saw best as the UI designer of that module ... and being a designer myself ... you know why i may have preferred a space before the colon.

I just feel foolish having to defend a 'space' ... on an relatively insignificant issue.

i don't know about us ... but the " **space** " before my colon sure must be feeling on top of the world ... as so many people are debating and discussing it !!! 

:)

Erick555 replied:
There aren't many people on the world who feel need to defend putting space before a colon when everyone recommend against it, so it's not surprising that people are linking this issue in various corners of the internet which is making you pretty famous 😄 .

aiamuzz replied:
@Erick555 ... wish it was me ... but my ... **space before the colon** ... that's actually become famous ... Damn !!! ... thanks to @AsciiWolf ... :cry:

aiamuzz committed:
Update README.md

aiamuzz committed:
Update README.md

I have clarified on why i use spaces ... but if its still so much of an issue that there is a space before the colon ... just let me know ... i'll remove them ... you know actually i have kind of started to get jealous of this 'space' before the colon ... shit its hogging all the limelight !!!

aiamuzz commented:
@alexlarsson ...

I was wondering if we had this type of readme infomation across all flathub application repo's ... I have added the website(flathub) link and the Runtime/Sdk info also there ... do you think it would be nice to have this info on all flathub application repo pages ?

If you give a nod ... i'll start publishing a README across all repo's one by one ...

**PS : Lets end the issue of 'the space before the colon' here once and for all ... i don't want it to get anymore popular beyond this PR ... its hogging all the attention !!!**

alexlarsson commented:
I think that is up to each individual maintainer. Having such info in a readme is one more place it can go stale and one more place you have to update it. Also, it seems more likely that you end up on the flathub page rather than the github one if you google for the app.

aiamuzz committed:
Update README.md

aiamuzz commented:
> I think that is up to each individual maintainer. Having such info in a readme is one more place it can go stale and one more place you have to update it. 

@alexlarsson ... hmmmmm ... never thought it that way makes sense though ...

Considering this then the runtime/sdk information is best left out ... as it would require to be updated ... so i have dropped it ...

The rest of the information in this commit ... is pretty much static ... requiring no update ... would you be fine to have this info on the flathub pages given its info about the app which otherwise is a blank page ?

>Also, it seems more likely that you end up on the flathub page rather than the github one if you google for the app.

that would actually be good ... don't you think ... if a user was looking to install a flathub'd version of a particular app he SHOULD be directed to the 'flathub page' and not the github page ... a normal user would only want to install that app and that information is served at flathub with a better UI/UX ... github is more of a nerdy hangout ... if google directed people to flathub more then i'd say, so be it !

Also subject to your approval of this ... i thought i'd suggest that we consider adding the github page Link on the flathub application page ... that way even the nerds could redirect from there to github.

plata commented:
@alexlarsson can we add the required extensions to the flathub page? If so, I would prefer if the README would only link there.

AsciiWolf commented:
Any news?

plata commented:
As you know, the help link will be pointing to our online documentation. Therefore I think it is sufficient to link to the flathub page here.

AsciiWolf commented:
Not sure what changed, but it looks like the 32-bit Compat runtime is now automatically installed during PlayOnLinux installation.

```
$ flatpak --user install flathub org.phoenicis.playonlinux
Looking for matches…

org.phoenicis.playonlinux permissions:
    ipc            network              x11      devices
    multiarch      file access [1]

    [1] xdg-documents, xdg-download


        ID                                   Arch   Branch Remote  Download
 1.     org.phoenicis.playonlinux            x86_64 stable flathub < 110.7 MB
 2.     org.freedesktop.Platform.Compat.i386 x86_64 18.08  flathub < 177.8 MB

Proceed with these changes to the user installation? [Y/n]: 
```

I am using Flatpak 1.2.4 on Fedora 29.

plata commented:
Ok, that's interesting. We didn't change anything from our side, so this must be something in flatpak. I didn't follow the flatpak changes closely therefore I'm not aware what might have changed. In any case, this probably means that this PR can be closed, right?

AsciiWolf commented:
Yep, seems to be ok to close this PR. :)
